### PR TITLE
Srinter 741

### PR DESCRIPTION
**Pull request by Chris7791**

### CODE REVIEW

File: config/sync/image.style.page_image_resonsive_retina_nocrop.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+uuid: 9032cc15-071a-4575-8959-34a31601fe62
+langcode: en
+status: true
+dependencies: {  }
+name: page_image_resonsive_retina_nocrop
+label: 'page image resonsive retina nocrop'
```
Comment (ooneill547) [inline config/sync/image.style.page_image_resonsive_retina_nocrop.yml]: Not important really but it looks like there is a spelling mistake here

**Review by omahm — COMMENTED** [review]
Looks good as as Orla pointed out there's a typo.

**Review by DuttonMa — CHANGES_REQUESTED** [review]
Looks great, just one thing that I noticed was that the 'Basic HTML' and 'Full HTML' text formats are only available to administrators, I think that at least the basic one should be available to pretty much everyone.
I also think that it is worth fixing the spelling mistake that Orla pointed out as it is just likely to lead to mistakes further down the line.

**Comment by DuttonMa** [comment]
Looks good to go now

**Pull request merged**